src/components/addToFavorite/AddToFavorite.tsx

```diff
@@ -0,0 +1,69 @@
+import { useContext, useEffect, useState } from 'react';
+import { GlobalContext } from '../../reducer';
+import { Product } from '../../types/product';
+
+import './addToFavorite.scss';
+
+type Props = {
+  product: Product
+};
+
+export const AddToFavorite:React.FC<Props> = ({ product }) => {
+  const [state, dispatch] = useContext(GlobalContext);
+  const [selectedLike, setSelectedLike] = useState(false);
+
+  const remove = () => {
+    const list: Product [] | [] = JSON
+      .parse(localStorage.getItem('likeList') as string) || [];
+
+    setSelectedLike(false);
+    if (list.length) {
+      localStorage
+        .setItem(
+          'likeList',
+          JSON.stringify(list
+            .filter((el: Product) => el.age !== product.age)),
+        );
```

Comment:
you can wrap logic with localStorage with some util and use it instead of add similar code many times

Reply:
+